pallets/investments/src/benchmarking.rs

```diff
@@ -56,6 +56,9 @@ mod benchmarks {
 
 	#[benchmark]
 	fn update_invest_order() -> Result<(), BenchmarkError> {
+		#[cfg(test)]
+		crate::mock::configure_accountant_mock();
```

Comment:
Completely astonished by the needed for this.

Without the above line, the test crashes because the mock closure methods are not defined. Nevertheless, `configure_accountant_mock()` is already called under `TextExternatilitiesBuilder::build()`, so I would expect to have already defined the mock methods. 

It seems like for benchmarking, somehow, the thread used to build the externality is not the same as the one used to benchmark. And then, the internal `thread_local` used by the mock builder doesn't correspond to the one used.

Reply:
Actually, for testing the 4 benchmarking, just one call to `TextExternatilitiesBuilder::build()` is done

Reply:
Interesting finding.